TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8299,6 +8299,11 @@ single_table_or_view_reference: tableOrViewOrCorrelationName correlation_clause?
 correlation_clause: SQL_AS? (correlation_name=UserDefinedWord) new_column_names?;
 new_column_names: LeftParenthesisSeparator new_column_name (SQL_CommaSeparator new_column_name)* RightParenthesisSeparator;
 new_column_name: UserDefinedWord;
+
+date: ({ string.Equals(CurrentToken.Text, "DATE", System.StringComparison.OrdinalIgnoreCase) }? KeywordDATE=UserDefinedWord);
+time: ({ string.Equals(CurrentToken.Text, "TIME", System.StringComparison.OrdinalIgnoreCase) }? KeywordTIME=UserDefinedWord);
+timestamp: ({ string.Equals(CurrentToken.Text, "TIMESTAMP", System.StringComparison.OrdinalIgnoreCase) }? KeywordTIMESTAMP=UserDefinedWord);
```

Comment:
Ok I was wrong because `DATE`, `TIME` and `TIMESTAMP` are not reserved words. So we actually need those and the definition for datetime_constant would be `datetime_constant: (date | time | timestamp) AlphanumericLiteral;`